pkg/apis/OWNERS

```diff
@@ -2,7 +2,14 @@
 
 approvers:
 - technical-oversight-committee
-- api-core-wg-leads
+- serving-wg-leads
```

Comment:
assuming wg leads here rather than writers since this would be for api changes...